cli/command/container/stop.go

```diff
@@ -56,12 +56,13 @@ func runStop(dockerCli *command.DockerCli, opts *stopOptions) error {
 	for _, container := range opts.containers {
 		if err := <-errChan; err != nil {
 			errs = append(errs, err.Error())
+			fmt.Fprintf(dockerCli.Err(), "%s\n", errs)
```

Comment:
Got it and  I added the duplicate logs here, because I studied startContainersWithoutAttachments in container/start.go.
Thanks for commands